src/plugins/console/server/routes/api/console/proxy/create_handler.ts

```diff
@@ -138,7 +138,8 @@ export const createHandler =
     for (let idx = 0; idx < hosts.length; ++idx) {
       const host = hosts[idx];
       try {
-        const uri = toURL(host, path);
+        const encodedPath = encodeURI(path);
```

Comment:
Nice catch 👍 

There is one small issue, we need to address before getting it ready to merge. 

From [the MSD docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/encodeURI#:~:text=Note%20that%20encodeURI()%20by%20itself%20cannot%20form%20proper%20HTTP%20GET%20and%20POST%20requests%2C) we can see, that `encodeURI` can't form the valid HTTP POST and GET requests. It is not escaping chars, like `; , / ? : @ & = + $`. The docs recommend us to use [`encodeURIComponent`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/encodeURIComponent) instead.

Not only `PUT` requests are passing through this function, but also `GET`. 

@mibragimov, what do you think about replacing `encodeURI` with `encodeURIComponent`?